CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
-## [1.4.0] &ndash; 2023-01-26
+## [1.4.1] &ndash; 2023-01-26
+
+### Fixed
+
+- Fixed an issue with the element fetching stuck in a loop after recovering from invalid credentials.
```

Comment:
Will this suggesting work, too?
```suggestion
- Fixed an issue with recursive element fetching after recovering from invalid credentials.
```

Reply:
```suggestion
- Fixed an issue with infinite element fetching after recovering from invalid credentials.
```
Just a small change, what do you think?

Reply:
Then I'll add `loop` to that wording so that you have:
```suggestion
- Fixed an issue with an infinite element-fetching loop after recovering from invalid credentials.
```

Reply:
Awesome, I will take the last one 😉